### PR TITLE
Add docs page with basic CLI and API usage

### PR DESCRIPTION
**Pull request by sebmarkbage**
Just lists the basic installation instructions, API usage and options.

### CODE REVIEW

**Comment by TheSavior** [comment]
Should this page be considered the "getting started" page?

Also, GitHub on my phone isn't letting me do inline comments. Will the .code::before selector conflict with the ones that exist right now that say input and output on the index page?

A screenshot would be valuable :+1:

**Comment by sebmarkbage** [comment]
Good call. I'll rename to Getting Started.

`.code` won't conflict because the existing ones use a different class name.

**Comment by TheSavior** [comment]
Can you add a link to the button group in the hero to this page as well? So it has both the getting started and try it out links there.

**Comment by TheSavior** [comment]
Also, .code seems too close to the code tag (which would conflict) for my taste. I'd prefer if it was something else but I wouldn't gate this change on that.

**Comment by sebmarkbage** [comment]
screencapture-file-users-sema-projects-prepack-website-getting-started-html-1493860497115

File: getting-started.html [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+          <p class="align-left">
+            For advanced uses <a href="#options">see the API options</a> or <code>prepack --help</code>.
+          </p>
+          <div class="code-block language-sh">
+            <pre><code>prepack [ --out output.js ] [ --compatibility jsc ] [ --mathRandomSeed seedvalue ] [ --srcmapIn inputMap ] [ --srcmapOut outputMap ] [ --speculate ] [ --trace ] [ -- | input.js ] [ --singlePass ] [ --debugNames ] [ --logStatistics ]</code></pre>
+          </div>
+
+          <h3>REPL</h3>
+          <p class="align-left">
+            You can also run Prepack in REPL mode. This probably isn't very useful but it lets you test out bugs Prepack's interpreter.
```
Comment (TheSavior) [inline getting-started.html]: Typo at the end of this line

**Pull request merged**